qutebrowser/browser/commands.py

```diff
@@ -1206,7 +1223,17 @@ def spawn(self, cmdline, userscript=False, verbose=False,
             cmd, args, userscript))
         if userscript:
             # ~ expansion is handled by the userscript module.
-            self._run_userscript(cmd, *args, verbose=verbose)
+            # dirty hack for async call because of:
+            # https://bugreports.qt.io/browse/QTBUG-53134
+            # until it fixed or blocked async call implemented:
+            # https://github.com/qutebrowser/qutebrowser/issues/3327
+            self.userscript_object = {
```

Comment:
Same as with `yank_object` - you should be able to pass those to `self._selection_callback` with `functools.partial` instead.